src/components/app/routes/loaders/rootLoader.ts

```diff
@@ -33,33 +35,56 @@ const makeRootLoader: Types.MakeRouteLoaderFunctionWithQueryClient = function ma
 
     // Retrieve linked enterprise customers for the current user from query cache
     // or fetch from the server if not available.
-    const enterpriseLearnerData = await queryClient.ensureQueryData<Types.EnterpriseLearnerData>(
-      queryEnterpriseLearner(username, enterpriseSlug),
+    const matchedBFFQuery = resolveBFFQuery(
+      requestUrl.pathname,
     );
+    let enterpriseLearnerData: Types.EnterpriseLearnerData | any;
```

Comment:
[curious] Why is this type using `any`? Including it largely defeats the purpose of including `Types.EnterpriseLearnerData`, since it forces `enterpriseLearnerData` to be `any` instead of the actual type, `EnterpriseLearnerData`.

Reply:
moot in refactor